drivers/gles3/rasterizer_scene_gles3.cpp

```diff
@@ -3970,7 +3974,8 @@ void RasterizerSceneGLES3::render_scene(const Transform& p_cam_transform,const C
 		_render_mrts(env,p_cam_projection);
 	}
 
-	glPolygonMode(GL_FRONT_AND_BACK,GL_FILL);
+// this is not supported on iOS ES3 anymore.. 
+//	glPolygonMode(GL_FRONT_AND_BACK,GL_FILL);
```

Comment:
hehehe, yes and no... The FRONT_AND_BACK polymode is GL_FILL by default. From OpenGL 1.0 you could switch it between GL_FILL, GL_POINT and GL_LINE and was often used to render stuff in wireframe. I suspect that this was the case for GODOT at some point in time as well. 

Ever since the fixed pipeline was replaced by programmable shaders however this is no longer used and for OpenGL ES seems to have long been the only option. There is no code anywhere in Godot that every switches this to anything else.

What is weird here is that in Apples ES3.h glPolygonMode is still available (even if it doesn't do anything) but GL_FILL seems to be missing. 

That all said, it explains why commenting out isn't a bad thing but in all truth and honesty, I just commented it out to test if it would finally compile and forgot to put a #ifndef IPHONE around it just in case :)